Makefile

```diff
@@ -6,6 +6,7 @@ BUILD_FLAGS := -ldflags="-w $(COMMON_FLAGS)"
 DEBUG_BUILD_FLAGS := -ldflags="$(COMMON_FLAGS)"
 FILES := odo dist
 TIMEOUT ?= 1800s
+SPEC_EXEC_METHOD ?=
```

Comment:
what this env variable do?
This should be explained in the comment

Reply:
Done